.github/workflows/nightly-automated-builds.yaml

```diff
@@ -0,0 +1,26 @@
+name: Nightly Automated Builds
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: "0 6 * * *" #  Every day at 6AM UTC which is 6PM NZDT
+
+jobs:
+  create-tags:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.PAT_WITH_WORKFLOW_SCOPE }}
```

Comment:
Needs to be created, the general github_token doesn't have the right permissions

Reply:
Have tested to just use the general github_token and works with it